anomalib/models/cflow/model.py

```diff
@@ -17,215 +17,17 @@
 # See the License for the specific language governing permissions
```

Comment:
I would personally prefer to rename `model.py` to `lightning_model.py` to match the pattern with `torch_model.py`